Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
I also like LaTeX. Our previous books were written in LaTeX but, we changed because:

1. LaTeX can't generate nice HTML
2. LaTeX is heavy to install so people tend not to contribute
3. LaTeX has much more features which makes it a bit harder to learn